core/core.cc

```diff
@@ -349,6 +363,69 @@ void TaskGraph::execute_point(long timestep, long point,
   k.execute();
 }
 
+void TaskGraph::execute_point(long timestep, long point,
```

Comment:
Instead of making a complete duplicate copy of this function, can we make the original version call the new one, but with NULL and 0 for scratch_ptr and scratch_bytes?